packages/components/src/SegmentedControl/SegmentedControlBase.tsx

```diff
@@ -6,26 +7,36 @@ import React, {
 } from "react";
 import styles from "./SegmentedControl.module.css";
 
-const SegmentedControlBase = forwardRef<HTMLDivElement, PropsWithChildren>(
-  function SegmentedControlBase({ children }, ref) {
-    const optionCount = Children.count(children);
+export interface SegmentedControlBaseProps extends PropsWithChildren {
```

Comment:
I don't _think_ we need to export this interface as they are only used here?

Reply:
This does need to be exported since it is used as part of a `forwardRef` in `SegmentedControlBase`